### DISCUSSION TITLE
How to tell if Postal is using spamassassin?

### DISCUSSION THREAD

**Discussion by Igcorreia** [postalserver/postal#3249] [Help with using Postal] [answered]
I have it running, port is the correct one. Updated config to use the correct port and ip. But postal doesn't check the spam.

I don't see any error on the logs. I am kind of stuck. Any ideas?

![image](https://github.com/user-attachments/assets/235625d2-21d8-42a8-96b8-a5810a7776ef)

**Comment by willpower232** [comment]
If you've restarted Postal after changing the config file, are you expecting it to scan messages you're sending or messages you're receiving? I think you have to go into the settings in the web UI to turn it on for one of those directions.

**Reply by willpower232** [reply]
re adding the screenshot for future lost souls

![image](https://github.com/user-attachments/assets/a3a36fe1-74ae-4241-9634-da4c4f48c619)

**Comment by Igcorreia** [comment]
@willpower232 The spam threshold is set by default. We just need to add the relevant settings to the postal.yml configuration file to enable it. Postal should automatically detect the settings and start using them.

When I send a test email via the CLI, I can confirm that SpamAssassin is working correctly. Additionally, I can verify that the Perl service is running on the correct port and internal IP.

I’m unsure where to debug this further. I want to check the logs to see if Postal is attempting to use SpamAssassin and whether any errors are being reported.

Service is running
![image](https://github.com/user-attachments/assets/4c50add3-d486-4749-8c3b-2a37a96306e6)

Correct Ports and IP
![image](https://github.com/user-attachments/assets/1ecd6656-6797-4e1b-8e06-2ff2303288f1)

Postal Config
![image](https://github.com/user-attachments/assets/b7a5fbb5-1aeb-4c8e-9e53-87e1b5690a5a)

Postal Stoppped
Postal Restarted

Email send via en external application using SMTP

and nothing :(

![image](https://github.com/user-attachments/assets/7095fa66-235f-4381-bcb0-2b4bdcc8c723)

**Reply by willpower232** [reply, marked as answer]
Are you expecting it to scan messages you are sending from Postal or messages being received by Postal?

Scanning for messages sent from Postal is off be default.

![image](https://github.com/user-attachments/assets/717649aa-12fe-4b45-95c3-e3db127041cc)

**Comment by Igcorreia** [comment]
@willpower232 I thought that messages SENT were also scanned. I will send it with the receiving ones and report back. Thanks.

**Comment by Igcorreia** [comment]
@willpower232 updating the OUTBOUND SPAM THRESHOLD had an immediate impact :) 
![image](https://github.com/user-attachments/assets/0af12d15-372f-429e-9cfa-2c18ccd83d4e)

Testing Inbound next.

**Comment by Igcorreia** [comment]
It works. Thank you @willpower232.
![image](https://github.com/user-attachments/assets/805ff607-2eff-4caf-a8a3-2a4fdc47637e)
